java-backend/src/main/java/com/project/spring/Endpoint/UnequalPayEndpoint.java

```diff
@@ -0,0 +1,43 @@
+package com.project.spring.Endpoint;
+
+import com.google.gson.Gson;
+import com.project.spring.Services.UnequalPayLookupService;
+import io.swagger.annotations.ApiOperation;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.ArrayList;
+
+@RestController
+@RequestMapping(value = "/api/unequalPayLookup")
+public class UnequalPayEndpoint {
+
+    @Autowired
+    private UnequalPayLookupService unequalPayLookupService;
+
+    final static Logger logger = LogManager.getLogger(UnequalPayEndpoint.class);
+
+    @ApiOperation(value = "lists all of the employees that make less than their juniors")
+    public ResponseEntity<String> findUnequallyPaidEmployees() {
+        logger.info("Handling request for list of employees that make less than their juniors");
+        ArrayList<Integer> unequalEmployees = unequalPayLookupService.findUnequallyPaidEmployees();
```

Comment:
Your response is essentially a list of integer id's, but the client wont ever know or care about those ids because that's something you maintain within your database. The ids themselves dont mean anything to anyone outside of the application, and so I recommend sending an object to the client that holds more meaningful data. Ex: names, salaries, etc.